sources/ssh_prompt/local

```diff
@@ -0,0 +1,74 @@
+[ ${SHELL##*/} = "zsh" ] && local GBT__WHICH_OPTS='-p'
+
+function gbt__err() { echo "$@" >&2; }
+
+function gbt_ssh() {
+    [ -z "$GBT__HOME$GBT__PROFILE" ] && gbt__err "'GBT__HOME' not defined" && return 1
+    [ -z "$GBT__HOME$GBT__THEME" ] && gbt__err "'GBT__HOME' not defined" && return 1
+    [ -z "$GBT__PROFILE" ] && GBT__PROFILE="$GBT__HOME/sources/ssh_prompt/remote"
+    [ -z "$GBT__THEME" ] && GBT__THEME="$GBT__HOME/themes/ssh_prompt"
+
+    local SSH_BIN=$(which ${GBT__WHICH_OPTS} ssh 2>/dev/null)
+    [ $? -ne 0 ] && gbt__err "'ssh' not found" && return 1
+
+    if [[ " ${GBT__SSH_IGNORE[*]} " == *" ${@[-1]} "* ]]; then
+        $SSH_BIN $@
+    else
+        $SSH_BIN -t $@ "
+            cat /etc/motd 2>/dev/null;
+            GBT__CONF=\"/tmp/.gbt.$RANDOM\" &&
+            echo \"$(base64 $GBT__PROFILE | tr -d '\r\n')\" | base64 -d > \$GBT__CONF &&
+            echo \"$(alias | awk '/gbt_/ {sub(/^(alias |)(gbt___|)/,"", $0); print "alias "$0}')\" >> \$GBT__CONF &&
```

Comment:
There is an error in `awk` regex. As of now it fails with following error:
```bash
$ gbt_ssh 192.168.0.1
awk: illegal primary in regular expression ^(alias |)(gbt___|) at (gbt___|)
 source line number 1
 context is
	/gbt_/ {sub(/^(alias |)(gbt___|)/,"", >>>  $0) <<<
```

Line should look like this:
```bash
echo \"$(alias | awk '/gbt_/ {sub(/^(alias )?(gbt___)?/,"", $0); print "alias "$0}')\" >> \$GBT__CONF &&
```
Can you remind me, where do we use `gbt___` prefix, as I couldn't find it in any existing configuration file?